### DISCUSSION TITLE
Status Bar

### DISCUSSION THREAD

**Discussion by wizmo2** [sle118/squeezelite-esp32#103] [Ideas]
I'd like to implement a simple status bar similar to the Radio, showing the play status icon, playlist control icons, time, battery level icon, and wifi signal icon.  It looks like the Radio has a reserved area at the bottom of the screen, which should be relatively easy to adapt to displayer.

@philippe44, I notice that you have been working on a battery_level_src (which I don't see exposed anywhere).  Is this something you have been looking into already?

**Comment by philippe44** [comment]
I've not looked into that but you would have to manage that from LMS, IMHO. There is no mechanism to manage "conflicts" at the device level nor things like viewport. I had to keep the display driver as simple as possible to keep CPU consumption "reasonable" (remember all the other things that poor esp32 has to do...)
